src/screens/LoginScreen.js

```diff
@@ -1,14 +1,97 @@
-import { StyleSheet, Text, View } from "react-native";
-import React from "react";
-
-const LoginScreen = () => {
-    return (
-        <View>
-            <Text>LoginScreen</Text>
-        </View>
-    );
+import {
+  StyleSheet,
+  Text,
+  SafeAreaView,
+  Button,
+  TextInput,
+  Alert
+} from "react-native";
+import { firebase } from "../firebase/Config";
+import { useState } from "react";
+
+const initialState = {
+  email: "",
+  password: ""
+};
+
+const LoginScreen = ({ navigation }) => {
+  const [credentials, setCredentials] = useState(initialState);
+
+  function handleChange(event, name) {
+    setCredentials({
+      ...credentials,
+      [name]: event.target.value
```

Comment:
Should be changed to "[name]: text" where text is the first parameter of the function.